src/nvim/option.c

```diff
@@ -2925,8 +2925,17 @@ static char *did_set_string_option(int opt_idx, char_u **varp, bool new_value_al
         || check_opt_strings(curbuf->b_p_bt, p_buftype_values, false) != OK) {
       errmsg = e_invarg;
     } else {
+      bool redraw_win = false;
```

Comment:
I think it doesn't hurt to just call `redraw_later()` twice, so this variable isn't really necessary.

Reply:
I changed how the PR triggers redraws so this part isn't relevant anymore